agenta-backend/agenta_backend/routers/app_router.py

```diff
@@ -71,22 +89,22 @@ async def list_app_variants(
         List[AppVariantOutput]: A list of app variants for the given app ID.
     """
     try:
-        user_org_data: dict = await get_user_and_org_id(request.state.user_id)
-
-        access_app = await check_access_to_app(
-            user_org_data=user_org_data, app_id=app_id
-        )
-        if not access_app:
-            error_msg = f"You cannot access app: {app_id}"
-            logger.error(error_msg)
-            return JSONResponse(
-                {"detail": error_msg},
-                status_code=403,
+        if FEATURE_FLAG in ["cloud", "ee"]:
```

Comment:
why an additional here ??

Reply:
I don't understand. It's not an additional if, it's the only if statement checking whether to perform rbac or not based on the feature flag.